### ISSUE TITLE
About TensorFlow, Scikit-learn, PyTorch module in ML

### ISSUE THREAD

**Issue by MrCodYrohit** [animator/learn-python#394]
I will add details about these 3 python libraries which are TensorFlow, Scikit-learn, PyTorch module in ML and explain it with the help of code snippets. Additionally I compare these libraries and try to generalize it when to use which libraries.

**Comment by animator** [comment]
Start with just one thing instead of a whole bunch of things.

**Comment by MrCodYrohit** [comment]
@animator Please assign this issue to me, and add the required GSSOC Label.

**Comment by MrCodYrohit** [comment]
> Start with just one thing instead of a whole bunch of things.

ok!
